src/extensions/default/analyzers/Microsoft.DotNet.UpgradeAssistant.Extensions.Default.CSharp.Analyzers/HtmlHelperAnalyzer.cs

```diff
@@ -91,5 +94,24 @@ private static string GetReplacementType(SyntaxNode node)
                 _ => AspNetCoreIHtmlHelperName // Default to IHtmlHelper
             };
         }
+
+        private static SyntaxNode GetQualifiedName(SyntaxNode name)
```

Comment:
this is now available as an extension method isn't it?

Reply:
Yep, good catch. I must have added this before extracting that into an extension method. I'll remove it from here.